src/store/projects/project-create-actions.ts

```diff
@@ -18,9 +19,33 @@ export interface ProjectCreateFormDialogData {
 
 export const PROJECT_CREATE_FORM_NAME = 'projectCreateFormName';
 
+export const isProjectRoute = ({ router }: RootState) => {
```

Comment:
Check file `routes/routes`. There are some methods for checking route. Maybe you could use some of them.